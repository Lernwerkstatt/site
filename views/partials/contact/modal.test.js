const modal = require("./modal.hbs");

describe("modal.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      modal({
        id: "testModal",
        headline: "I'm a headline",
        text:
          "Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates."
      })
    ).toMatchSnapshot();
  });
});
