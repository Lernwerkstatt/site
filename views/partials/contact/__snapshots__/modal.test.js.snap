// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal.hbs should match the snapshot 1`] = `
"<div class=\\"modal fade\\" id=\\"testModal\\" tabindex=\\"-1\\" role=\\"dialog\\" aria-labelledby=\\"testModal\\" aria-hidden=\\"true\\">
  <div class=\\"modal-dialog\\" role=\\"document\\">
    <div class=\\"modal-content\\">
      <div class=\\"modal-header\\">
        <h5 class=\\"modal-title\\" id=\\"testModal\\">Terms
          &amp; Conditions</h5>
        <button type=\\"button\\" class=\\"close\\" data-dismiss=\\"modal\\" aria-label=\\"Close\\">
          <span aria-hidden=\\"true\\">&times;</span>
        </button>
      </div>
      <div class=\\"modal-body\\">
        <p>I&#x27;m a headline</p>
        <pre style=\\"white-space: pre-wrap;\\">Lorem ipsum dolor sit amet consectetur adipisicing elit. Veritatis ea consequatur quibusdam sint laboriosam voluptates.</pre>
      </div>
      <div class=\\"modal-footer\\">
        <button type=\\"button\\" class=\\"btn btn-secondary\\" data-dismiss=\\"modal\\">Close</button>
      </div>
    </div>
  </div>
</div>
"
`;
